e2e/cypress/integration/plugins/marketplace/invalid_marketplace_url_spec.js

```diff
@@ -0,0 +1,95 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+// ***************************************************************
+// - [#] indicates a test step (e.g. # Go to a page)
+// - [*] indicates an assertion (e.g. * Check the title)
+// - Use element ID when selecting an element. Create one if none.
+// ***************************************************************
+
+// Group: @not_cloud @plugin_marketplace @plugin
+
+import * as TIMEOUTS from '../../../fixtures/timeouts';
+import {githubPlugin} from '../../../utils/plugins';
+
+describe('Plugin Marketplace', () => {
+    let townsquareLink;
+
+    before(() => {
+        cy.shouldNotRunOnCloudEdition();
+        cy.shouldHavePluginUploadEnabled();
+
+        cy.apiInitSetup().then(({team}) => {
+            townsquareLink = `/${team.name}/channels/town-square`;
+        });
+    });
+
+    beforeEach(() => {
+        // # Login as sysadmin
+        cy.apiAdminLogin();
+
+        // # Enable Plugin Marketplace and Remote Marketplace
+        cy.apiUpdateConfig({
+            PluginSettings: {
+                Enable: true,
+                EnableMarketplace: true,
+                EnableRemoteMarketplace: true,
+                MarketplaceUrl: 'example.com',
+            },
+        });
+
+        // # Cleanup installed plugins
+        cy.apiUninstallAllPlugins();
+
+        // # Visit the Town Square channel
+        cy.visit(townsquareLink);
+
+        cy.wait(TIMEOUTS.HALF_SEC).get('#lhsHeader').should('be.visible').within(() => {
+            // # Click hamburger main menu
+            cy.get('#sidebarHeaderDropdownButton').click();
+
+            // * Verify dropdown menu should be visible
+            cy.get('.dropdown-menu').should('be.visible').within(() => {
+                // * Verify Plugin Marketplace button should be visible then click
+                cy.findByText('Marketplace').should('be.visible').click();
+            });
+        });
+    });
+
+    it('MM-T1966 render an error bar', () => {
```

Comment:
This might be a different test since T1966 specifically tests the marketplace not being available with invalid URL.

Reply:
It was updated by [this previous change](https://github.com/mattermost/mattermost-webapp/pull/7335/files#diff-eff4c2c63102776a9928edfcef3e5f8ab844e6400215bcff1239b6d1803fb01eR178). I honestly didn't check it in details as I'm specifically fixing plugin related issue but looking at the test case itself, seems to me that it matched the written test case. Maybe just need to exactly match the error message of "Error connecting to the marketplace server. Please check your settings in the System Console.”.
But I'm good in removing it as you're more familiar with Marketplace test cases.

@josephbaylon Thoughts on this before I remove the test key as suggested above.

Reply:
I went ahead and applied the comment from Prapti as I needed to merge this and resolve the conflict from other PR.